ansible_image_validation/validation-playbooks/per-vm-validation.yaml

```diff
@@ -58,4 +58,22 @@
 - name: Check if blacklisted drivers are blacklisted by modprobe service
   include_tasks: validation-playbooks/blacklisted_drivers_validation.yaml
   ignore_errors: yes
-  when: ansible_os_family == "RedHat"
\ No newline at end of file
+  when: ansible_os_family == "RedHat"
+
+- name: Check if the nvme and pci drivers are present in all images
+  shell: lsinitrd /boot/initramfs-{{ ansible_facts.kernel }}.img | grep {{ item }}
+  register: check_nvme_pci_allimages
+  with_items:
+    - nvme
+    - pci    
+
+- debug:
+    var: check_nvme_pci_allimages.results  
+
+- name: "Write to error msg if some drivers are not present"
+  lineinfile:
+    path: "{{err_folder}}/err_msgs.log"
+    line: "\n Errors Detected as shown below: \n {{ check_nvme_pci_allimages.results }} "
```

Comment:
Be specific here Navpreet, 
NVME validation failed since <nvme> driver is not present.
NVME validation failed since <pci> driver is not present.